types/hawk/OTHER_FILES.txt

```diff
@@ -0,0 +1 @@
+lib/browser.d.ts
```

Comment:
I feel like I'm missing something obvious, but that's this file for?

Reply:
It's for using `hawk` in a browser context.

Reply:
Sorry, I'm still not getting it.  I'm used to seeing `export as namespace` when a package is also consumed in browsers.  Could you possibly provide an example usage showing how these types would be consumed?

Reply:
The package can both be used in Node.js and in the browser.
For Node.js, we export `{ server, client, crypto, utils }`, but not the content from `browser.d.ts`.

With the help from `OTHER_FILES.txt` (as described in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/#other_filestxt)), we are shipping the `browser.d.ts` file without the need to have it exported. And since this file uses `declare namespace hawk`, we can use `hawk.*` in the browser.

Reply:
By the way, the file was copied from the [`hapi__hawk`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/1d9bc329b0285c69420dfbd8ab59bf1eeb86fcf5/types/hapi__hawk) directory and it was working quite well :)

Reply:
If you're happy, I'm happy.